src/erlcloud_as.erl

```diff
@@ -367,6 +373,44 @@ resume_processes(GroupName, ScalingProcesses, Config) ->
     end.
 
 
+%% --------------------------------------------------------------------
+%% @doc Detach the given instances from the group using the default configuration
+%% without decrementing the desired capacity of the group.
+%% @end
+%% --------------------------------------------------------------------
+-spec detach_instances(list(string()),string()) -> aws_autoscaling_activity().
+detach_instances(InstanceIds, GroupName) ->
+    detach_instances(InstanceIds, GroupName, erlcloud_aws:default_config()).
+
+%% --------------------------------------------------------------------
+%% @doc Detach the given instances from the group.  The 3rd parameter can be:
+%% 'false' to not decrement the desired capacity of the group
+%% 'true' to decrement the desired capacity of the group
+%% Config a supplied AWS configuration.
+%% @end
+%% --------------------------------------------------------------------
+detach_instances(InstanceIds, GroupName, false) ->
```

Comment:
Could be worth it, although according to API doc it is the only parameter there currently is. And it's the main use case, i.e. detach instance X from group Y and decrease or don't decrease capacity - could make an inner function accepting a list of params instead. Your thoughts?